handlers/goodshit.js

```diff
@@ -1,55 +1,106 @@
-var _      = require('lodash');
-var log4js = require('log4js');
-var format = require('format');
+const _			= require('lodash');
+const log4js = require('log4js');
+const format = require('format');
 
-var LOG = log4js.getLogger('goodshit');
+const LOG = log4js.getLogger('goodshit');
 
 module.exports = {
-  name: 'goodshit',
-  command: ':gs',
-  handler: goodshit
+	name: 'goodshit',
```

Comment:
what kind of goddamn circus you running with TABS

we are a spaces only kind of repo